release-notes/3.0/preview/api-diff/preview7/standalone-packages/3.0-preview7-standalone-packages_System.Net.Http.md

```diff
@@ -0,0 +1,20 @@
+# System.Net.Http
+
+``` diff
+ namespace System.Net.Http {
+-    public sealed class Utf8StringContent : HttpContent {
```

Comment:
Similarly, maybe this file should be removed. It is part of the experimental NuGet package. It was never added to a previous API diff.

https://github.com/dotnet/core/search?utf8=%E2%9C%93&q=Utf8StringContent&type=

Reply:
Agreed